index.html

```diff
@@ -514,43 +523,66 @@
         </dt>
       </dl>
       <p>
-        A <code>PermissionStatus</code> instance has an associated
-        <var>permission</var> that is a <a>PermissionDescriptor</a>.
+        <code>PermissionStatus</code> instances are created with the following
```

Comment:
I'm curious why it's useful to define this as an internal slot as opposed to just saying that it has an associated PermissionDescriptor? It seems a bit more confusing to me.